types/react-native/index.d.ts

```diff
@@ -10,6 +10,7 @@
 //                 Manuel Alabor <https://github.com/swissmanu>
 //                 Michele Bombardi <https://github.com/bm-software>
 //                 Tanguy Krotoff <https://github.com/tkrotoff>
+//                 Alexander T. <https://github.com/a-tarasyuk>
```

Comment:
Are you aware that this means you’re signing up to be a maintainer and that you’ll be asked to review new PRs to these typings?

Reply:
Yes I am aware, should I remove it?

Reply:
Oh no, if you’re really up for doing that then by all means! I’m asking because sometimes people mistakingly think this is a list of people that have made changes to the file.

Cool, welcome to the club then 🙏